StitchCore-iOS/StitchCore-iOS/Core/Auth/Providers/UserPassword/UserPasswordAuthProvider.swift

```diff
@@ -88,45 +99,54 @@ public protocol UserPasswordAuthProviderClient {
      *     - completionHandler: The handler to be executed when the request is complete.
      */
     func reset(password: String, withToken token: String, withTokenId tokenId: String, completionHandler: @escaping (Error?) -> Void)
+
+    // swiftlint:enable line_length
 }
 
 private class UserPasswordAuthProviderClientImpl: CoreUserPasswordAuthProviderClient, UserPasswordAuthProviderClient {
     private let dispatcher: OperationDispatcher
-    
+
     init(withRequestClient requestClient: StitchRequestClient,
          withRoutes routes: StitchAuthRoutes,
          withDispatcher dispatcher: OperationDispatcher) {
         self.dispatcher = dispatcher
         super.init(withRequestClient: requestClient, withRoutes: routes)
     }
-    
-    func register(withEmail email: String, withPassword password: String, completionHandler: @escaping (Error?) -> Void) {
+
+    func register(withEmail email: String,
+                  withPassword password: String,
+                  completionHandler: @escaping (Error?) -> Void) {
         dispatcher.run(withCompletionHandler: completionHandler) {
-            let _ = try super.register(withEmail: email, withPassword: password)
+            _ = try super.register(withEmail: email, withPassword: password)
         }
     }
-    
-    func confirmUser(withToken token: String, withTokenId tokenId: String, completionHandler: @escaping (Error?) -> Void) {
+
+    func confirmUser(withToken token: String,
+                     withTokenId tokenId: String,
+                     completionHandler: @escaping (Error?) -> Void) {
         dispatcher.run(withCompletionHandler: completionHandler) {
-            let _ = try super.confirmUser(withToken: token, withTokenId: tokenId)
+            _ = try super.confirmUser(withToken: token, withTokenId: tokenId)
```

Comment:
[nit][q] Do we need the `_ =` syntax here? Without a return value, there shouldn't be a warning here.

Reply:
these methods in super do have a return value (the raw `Response`), so we have to explicitly ignore it. (and it will properly throw when the response is not a 200-299)